docs/man_pages/project/testing/build-android.md

```diff
@@ -26,7 +26,13 @@ General | `$ tns build android [--compileSdk <API Level>] [--key-store-path <Fil
 * `--key-store-alias-password` - Provides the password for the alias specified with `--key-store-alias-password`. You can use the `--key-store-*` options along with `--release` to produce a signed release build. You need to specify all `--key-store-*` options.
 * `--copy-to` - Specifies the file path where the built `.apk` will be copied. If it points to a non-existent directory path, it will be created. If the specified value is existing directory, the original file name will be used.
 * `--bundle` - Specifies that the `webpack` bundler will be used to bundle the application.
-* `--env.*` - Specifies additional flags that the bundler may process. May be passed multiple times. For example: `--env.uglify --env.snapshot`.
+* `--env.*` - Specifies additional flags that the bundler may process. May be passed multiple times. Supported additional flags:
+    *   `--env.aot` - creates Ahead-Of-Time build (Angular only)
+    *   `--env.snapshot`- creates Snapshot (only on Mac OS & only for Android)
```

Comment:
We could link the docs article when the help is read in the HTML version (https://docs.nativescript.org/performance-optimizations/bundling-with-webpack#v8-heap-snapshot). Maybe we could also mention the faster startup time.